frontend/packages/dev-console/src/components/monitoring/overview/MonitoringTab.tsx

```diff
@@ -1,13 +1,35 @@
 import * as React from 'react';
 import { OverviewItem } from '@console/shared';
-import MonitoringOverview from './MonitoringOverview';
+import { Firehose } from '@console/internal/components/utils';
+import MonitoringOverviewWrapper from './MonitoringOverviewWrapper';
 
 type MonitoringTabProps = {
   item: OverviewItem;
 };
 
-const MonitoringTab: React.FC<MonitoringTabProps> = ({ item: { obj: res, events } }) => {
-  return <MonitoringOverview resource={res} events={events} />;
+const MonitoringTab: React.FC<MonitoringTabProps> = ({ item }) => {
+  const { uid, name, namespace } = item?.obj?.metadata;
+  const kind = 'Pod';
```

Comment:
nit: can import `import { PodModel } from '@console/internal/models';` and use `PodModel.kind`

Reply:
i think it'll be `pod` as we are fetching events for pods irrespective of workloads i.e d,dc etc